src/Hl7.Fhir.Shims.Base/Model/ValueSetExpansionExtensions.cs

```diff
@@ -67,27 +67,23 @@ internal static List<CSDC> RemoveCode(this ICollection<CSDC> concepts, string co
 
         private static ICollection<CSDC> removeCodeByPredicate(this ICollection<CSDC> concepts, Predicate<CSDC> predicate)
```

Comment:
Right now we have side effects: we remove items, but we return the collection because we want to keep the fluent syntax. We should really be making this pure, that way the signature makes sense.

Reply:
You are suggesting to make this a void you mean?